tests/it/query.rs

```diff
@@ -106,6 +106,16 @@ async fn long_query() {
         .unwrap();
 
     assert_eq!(got_string, long_string);
+
+    let got_string = client
+        .query("select ?")
+        .bind(&long_string)
+        .with_option("readonly", "2")
+        .fetch_one::<String>()
+        .await
+        .unwrap();
+
+    assert_eq!(got_string, long_string);
```

Comment:
ideally assert would be that readonly came through as 2. Changing query to `show readonly` could have that effect